lib/vis_milk2/ns-eel2/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+project(nseel LANGUAGES C ASM_NASM)
```

Comment:
what this file for?

Reply:
I used it to build only nseel when testing. Isn't really needed for anything and can be removed.